source/platforms/databricks.md

```diff
@@ -1,12 +1,42 @@
 # Databricks
 
-## Databricks Notebooks
-
 You can install RAPIDS libraries into a Databricks GPU Notebook environment.
 
-### Launch a single-node Databricks cluster
+## DASK Rapids in Databricks MNMG Cluster
+
+You can launch Dask RAPIDS cluster on a multi-node GPU Databricks cluster. To do this, you must first create an [initialization script](https://docs.databricks.com/en/init-scripts/index.html) to install Dask before launching the Databricks cluster.
+
+Databricks recommends storing all cluster-scoped init scripts using workspace files. Each user has a Home directory configured under the `/Users` directory in the workspace. Navigate to your home directory in the UI and select **Create** > **File** from the menu, create an `init.sh` script with contents:
+
+```bash
+#!/bin/bash
+set -e
+
+# The Databricks Python directory isn't on the path in
+# databricksruntime/gpu-tensorflow:cuda11.8 for some reason
+export PATH="/databricks/python/bin:$PATH"
+
+# Install RAPIDS (cudf & dask-cudf) and dask-databricks
+/databricks/python/bin/pip install --extra-index-url=https://pypi.nvidia.com \
+      bokeh==3.2.2 \
```

Comment:
I pushed `dask-databricks` `0.2.0` earlier which pins bokeh. So we can remove this from the docs here.